SubmitFileGSD.sh

```diff
@@ -48,7 +48,10 @@ if [ ${localFlag} == "True" ]
         cp *DQM*.root ${curDir}/${outDir}/DQM/
     fi
   else
-    xrdcp -N -v *${dataTier}*.root root://eoscms.cern.ch/${eosArea}/${outDir}/${dataTier}/
+    a=`ls *.root`
```

Comment:
it's good keeping generic about the local file

we should foresee the option that there's more than 1 file, e.g. when there's RECO and DQM...